src/testapp/scenarios.cpp

```diff
@@ -370,6 +370,101 @@ static ScenarioMap_t make_scenarios()
         return setup_renderer;
     });
 
+    add_scenario("lander", "Lander simulation game", [] (TestApp& rTestApp) -> RendererSetupFunc_t {
```

Comment:
you could move your scenerio function to a new file, and call it from this file, if you want.